runtime/browser/lib/suggestions-element.js

```diff
@@ -53,18 +53,19 @@ class SuggestionsElement extends HTMLElement {
       this._root = this.attachShadow({mode: 'open'});
       this._root.appendChild(document.importNode(template.content, true));
       this.toast = this._root.querySelector('x-toast');
+      // TODO: regenerate all descriptions every time the toast is reopened.
       this.container = this._root.querySelector('suggestions');
     }
   }
 
-  addSuggestion({plan, descriptionGenerator, rank, hash}, index) {
+  addSuggestion({plan, description, rank, hash}, index) {
     let model = {
       index,
-      innerHTML: descriptionGenerator.description,
+      innerHTML: description,
```

Comment:
We should make sure and sanitize description strings that come out of manifests. We use `innerHTML` here to allow Arcs itself to inject rich-text (from view descriptions), but manifests should never do that, and the manifest text should be cleaned. A simple form of sanitization to start might be `str = str.replace(/</g, '&lt;')`

Reply:
done